src/holidays/united-states/federal-reserve-bank/is-federal-reserve-bank-holiday.spec.ts

```diff
@@ -0,0 +1,212 @@
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 2 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 9 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 16 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 23 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 30 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 6 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 13 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 20 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 27 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 4 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 11 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 18 }), true);
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 25 }), true);
+  });
+
+  it('works for all federal reserve bank holidays in 2022', () => {
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 0, date: 1 }), true); // New Year's Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 0, date: 17 }), true); // Birthday of Martin Luther King, Jr.
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 1, date: 21 }), true); // Washington's Birthday
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 4, date: 30 }), true); // Memorial Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 5, date: 19 }), true); // Juneteenth
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 6, date: 4 }), true); // Independence Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 8, date: 5 }), true); // Labor Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 9, date: 10 }), true); // Columbus Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 11 }), true); // Veteran's Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 10, date: 24 }), true); // Thanksgiving Day
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 25 }), true); // Christmas
+  });
+
+  it('works for all federal reserve bank holidays observed in 2022', () => {
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 5, date: 20 }), true); // Juneteenth Observed
+    assert.equal(isFederalReserveBankHoliday({ year: 2022, month: 11, date: 26 }), true); // Christmas Observed
+  });
+
+  it('works for all federal reserve bank holidays in 2023', () => {
+    assert.equal(isFederalReserveBankHoliday({ year: 2023, month: 0, date: 1 }), true); // New Year's Day
```

Comment:
as a philosophical question, should a federal reserve bank holiday that falls on a Sunday be classified as a holiday if it's actually observed the next day? i know this won't be a day in which the nacha files are generated but isn't that because it's a Sunday and not because it's a federal reserve bank holiday? not sure whether it's wrong or not, it just seems odd that this api returns 2 federal reserve bank holidays for Jan 1.

Reply:
good question @bobwieler. my thought here is, US Federal Reserve can be closed on a particular day because 1) that day is a weekend, 2) it's a federal holiday, and 3) a day on which a federal holiday is observed because the actual day falls on a weekend. So, I think, this interface should account for all those three scenarios.

Reply:
that's a good point and I like this suggestion. Will rename this file to `is-federal-reserve-bank-closed.ts`?

Reply:
works for me.